proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -30,24 +30,54 @@ service Msg {
   // Undelegate defines a method for performing an undelegation from a
   // delegate and a validator.
   rpc Undelegate(MsgUndelegate) returns (MsgUndelegateResponse);
+
+  // UnbondValidator defines a method for performing the status transition for a validator
+  // from bonded to unbonded
+  rpc UnbondValidator(MsgUnbondValidator) returns (MsgUnbondValidatorResponse);
```

Comment:
This skips unbonding?

Reply:
it's just a typo in the comment. resolved [[here](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/cbd11e9696914179859b19beb884e39070556205)]